erpnext/accounts/party.py

```diff
@@ -308,3 +308,13 @@ def set_taxes(party, party_type, posting_date, company, customer_group=None, sup
 		args.update({"use_for_shopping_cart": use_for_shopping_cart})
 
 	return get_tax_template(posting_date, args)
+
+def validate_party_frozen_disabled(party, party_type):
+	if party_type and party:
+		frozen_accounts_modifier = frappe.db.get_value( 'Accounts Settings', None,'frozen_accounts_modifier')
+		if not frozen_accounts_modifier in frappe.get_roles():
+			if frappe.db.get_value(party_type, party, "is_frozen"):
```

Comment:
@anandpdoshi Can we use something else as variable name instead of 'party' to avoid confusion and any code mishaps in future.